atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsGetCandidateCellsForSweepingBenchmarks.java

```diff
@@ -98,7 +96,7 @@ private int fullTableScan(TableReference tableRef,
                     .batchSizeHint(1000)
                     .maxTimestampExclusive(Long.MAX_VALUE)
                     .shouldCheckIfLatestValueIsEmpty(thorough)
-                    .timestampsToIgnore(thorough ? ImmutableSet.of() : ImmutableSet.of(Value.INVALID_VALUE_TIMESTAMP))
+                    .ignoreGarbageCollectionSentinels(!thorough)
```

Comment:
Will change this to a better name, since ignore can either mean ignore + delete them or leave them behind while sweeping. As is, I don't think this is very intuitive.